### ISSUE TITLE
Errors when running the version installed via npm

### ISSUE THREAD

**Issue by stevegrunwell** [10up/generator-wp-make#67]
There may be a discrepancy in [what's tagged for release 0.4.0 on npm](https://www.npmjs.com/package/generator-wp-make) and what's currently in master (though GitHub is not showing one):

I was trying to get `wp-make` working today and kept running into issues (npm 3.6.0, yo 1.7.0, `wp-make` installed via `npm install -g generator-wp-make`).

**List of generators (with blank space below wp-make):**

``` sh
$ yo --help
# ...
Available Generators:

  reveal
    slide
  vvv
    bootstrap
    json
  wp-make

  hubot
    script
```

**Help for the `wp-make` generator:**

``` sh
$ yo wp-make --help
Usage:
  yo wp-make:app [options]

Options:
  -h,   --help          # Print the generator's options and usage
        --skip-cache    # Do not remember prompt answers             Default: false
        --skip-install  # Do not automatically install dependencies  Default: false
```

**Following the instructions provided in the previous help menu:**

``` sh
$ yo wp-make:app
Invoke a subgenerator to get started!
Available Modules:
    yo wp-make:plugin
    yo wp-make:theme
    yo wp-make:child-theme
    yo wp-make:library
```

**Following the documentation found elsewhere for generating a new theme:**

``` sh
$ yo wp-make:theme
Error wp-make:theme

You don't seem to have a generator with the name wp-make:theme installed.
You can see available generators with npm search yeoman-generator and then install them with npm install [name].
To see the 8 registered generators run yo with the `--help` option.
```

Once I manually cloned and `npm link`d everything, the generator began behaving as expected. Is there a reason why this can't be installed via npm?

**Comment by lkwdwrd** [comment]
NPM was one commit behind. I also rolled in the bracket fixes from @jtsternberg and released to NPM as 0.4.1. Can you give it another test via NPM and make sure it's working for you? Seems to be ok for me now.

**Comment by lkwdwrd** [comment]
Going to close this for now as it appears to be working fine now.
